classes/custompage.php

```diff
@@ -98,10 +94,22 @@ public function update($data) {
      * @param string $item
      * @return mixed
      */
-    public function __get($item) {
+    public function __get($item): mixed {
```

Comment:
Refactor the name of ``$item`` to ``$name``
The method that implemented super class must have the same signature as parent method.

If you have question about which type  you have to use then you can visit https://www.php.net/manual/en/language.oop5.overloading.php#object.set

PS: Remember to rename variable in the method as well.
```suggestion
    public function __get(string $name): mixed {
```